homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
I propose someone with isy944 device make the fix then. Even if the current state of events is broken I'm afraid to break it further since I can't really test it.